salt/modules/dockercompose.py

```diff
@@ -330,13 +340,16 @@ def __load_project_from_file_path(file_path):
     :param path:
     :return:
     """
-    try:
-        project = get_project(
-            project_dir=os.path.dirname(file_path),
-            config_path=[os.path.basename(file_path)],
-        )
-    except Exception as inst:  # pylint: disable=broad-except
-        return __handle_except(inst)
+    if HAS_PYTHON_ON_WHALES:
+        project = DockerClient(compose_files=[file_path])
```

Comment:
Maybe we should consider later allowing multiple docker compose files.

Reply:
out of interest what would this gain. the module calls happen right now based on a specific filename / folder thats being called. 
that would seem like a change from existing functionality. 

my goal for this PR is to replicate existing functions only. that makes this easier to get merged in / tested / approved as a deprecation of the existing import

Reply:
Yeah, I only wanted to point it out. I needed the functionality in the past. But youre right that is more a topic for an own PR / Issue. (https://docs.docker.com/compose/extends/)